pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go

```diff
@@ -153,50 +168,75 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	return nil
 }
 
-func (r *ReconcileNvmeOfStorage) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
-	logger.Debugf("reconciling NvmeOfStorage. Request.Namespace: %s, Request.Name: %s", request.Namespace, request.Name)
+func (r *ReconcileNvmeOfStorage) getSystemEvent(e string) ControllerState {
+	if strings.Contains(e, "nvmeofstorage") {
+		return CR_UPDATED
+	} else if strings.Contains(e, "rook-ceph-osd") {
+		return OSD_STATE_CHANGED
+	}
+	panic("wrong event type")
```

Comment:
확장성 고려해서 switch case로 하면 어떨까요?

```suggestion
	switch {
	case strings.Contains(e, "nvmeofstorage"):
		return CR_UPDATED
	case strings.Contains(e, "rook-ceph-osd"):
		return OSD_STATE_CHANGED
	default:
		panic("wrong event type")
}
```

Reply:
지금은 의미가 없고 나중에 strings.contains 이 부분까지 변경하여 한번에 업데이트 필요해 보입니다.